desktops/kde.txt

```diff
@@ -130,28 +284,55 @@ if you know you will not need bluetooth, you have no need for bluedevil.
     +--------------------------------------------------------------------------+
 
     Make sure you do not currently have qt5 installed. The build files have been
-    tweaked for this repository to make use of dbus and eudev. As a result, 
-    carrying over the community version of these packages will lead to unknown 
-    problems. If you would like to skip building them and instead install the
-    prebuilt packages provided on the GitHub page, install the dependencies 
-    and do
+    tweaked for this repository to make use of dbus and libudev-zero. As a 
+    result, carrying over the community version of these packages will lead to 
+    unknown problems.
+
+    Because of the time required to build qt5* packages, some prebuilt tarballs
+    are available on the Github repository [5]. They are built in a standard 
+    KISS tarball, with C(XX)FLAGS=-march=-x86-64 -mtune=generic -Os -pipe.
+
+    NOTE: In general, it is a security risk to install prebuilt packages. these
+    archives are provided as a courtesy by $/dilyn-corner. Always verify the
+    authenticity of packages, and the identities of packagers!
+    
+    The current release is 5.15.1, available with the 2020.10-1 release. 
 
     +--------------------------------------------------------------------------+
     |                                                                          |
-    |   $ ver=2020.08                                                          |
-    |   $ qtver=5.15.0-1                                                       | 
+    |   $ ver=2020.10-1                                                        |
+    |   $ qtver=5.15.1-1                                                       | 
     |   $ url=https://github.com/dilyn-corner/KISS-kde/releases/download/$ver  |
     |                                                                          |
     |   $ wget $url/qt5.$qtver.tar.gz \                                        |
-    |          -O qt5\#$qtver.tar.gz                                           |
+    |          -O qt5@$qtver.tar.gz                                            |
     |   $ wget $url/qt5-declarative.$qtver.tar.gz \                            |
-    |         -O qt5-declarative\#$qtver.tar.gz                                |
+    |         -O qt5-declarative@$qtver.tar.gz                                 |
+    |                                                                          |
+    |   # Not a build-time requirement, but useful for e.g. falkon             |
     |   $ wget $url/qt5-webengine.$qtver.tar.gz \                              |
-    |          -O qt5-webengine\#$qtver.tar.gz                                 |
+    |          -O qt5-webengine@$qtver.tar.gz                                  |
```

Comment:
`@` as a separator is not yet in a full package manager release. To speed things up I may backport some things to a new 5.X.X release.

Reply:
I'm just being ready for the change :)

Reply:
Change has been made.